web/src/components/vis/mixins/axisPlot.js

```diff
@@ -44,6 +42,12 @@ export const axisPlot = {
       const { height, margin } = this;
       return height - margin.top - margin.bottom;
     },
+    xrange() {
+      return [-1, 1];
+    },
+    yrange() {
+      return [-1, 1];
+    },
```

Comment:
Gotcha. Any computed property that returns a constant value probably warrants a short comment to explain so people don't move it back to `data` in a future PR years from now.

Reply:
Added a comment (1604fb9). I left it out of `LoadingsPlot` because if anyone changes that, they'll get an error directing them to `axisPlot` in any case.